gst-plugin/src/drpai-models/drpai-tvm/rzv_drp-ai_tvm

```diff
@@ -0,0 +1 @@
+Subproject commit 065dcac9a73fffb8e75b1b7a498b2cf44e8a2b0b
```

Comment:
This addition should not be present at all. Please delete the whole `gst-plugin/src/drpai-models/drpai-tvm` directory.